.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
I think it's better to exclude all file patterns that are specific to your workstation and your tools once in `$HOME/.gitignore`, instead of adding noise in every single shared repo.

Reply:
Done!